### DISCUSSION TITLE
v2.26.0

### DISCUSSION THREAD

**Discussion by tyrasd** [openstreetmap/iD#9759] [Announcements]
After a little while, the newest release of iD contains some long awaited bug fixes and also some notable new features, this time mostly focused around various tagging topics. Let's take a look of a few highlights:

## Release News

* Fields for "multi" tags with `yes/no` values now also display the `no` state and allow to toggle between the two states (#9656)
  ![image](https://github.com/openstreetmap/iD/assets/1927298/07b73851-88c8-4b78-b61d-7ed255e466e6)
* The Address field now supports the `addr:place` tag (as an alternative to `addr:street`), this functionality is only active in selected countries, where this addressing schema is used (#9603)
  [![image](https://user-images.githubusercontent.com/1927298/235221474-239cc71d-fd2e-48bf-9cf4-73f25138a666.png)](https://ideditor-release.netlify.app//#background=Bing&disable_features=boundaries&id=w151682818&map=20.00/49.48880/8.46379)
* Support input fields with multiple tag keys: one main key and an optional list of _alternative_ keys, e.g. `email` / `contact:email` (https://github.com/openstreetmap/id-tagging-schema/issues/905)
* Show localized numbers in numeric fields (#8769)
  <img src="https://user-images.githubusercontent.com/1231218/138837083-1a4852ae-9b27-4523-b0c6-e8ba4df34ddd.png" width="359" alt=""> <i>(field in Vietnamese locale)</i>
  <img src="https://user-images.githubusercontent.com/1231218/138837128-a3901be9-38b6-46b2-ac73-76114612e1e7.png" width="359" alt=""> <i>(field in Arabic locale)</i>
* The Bing Streetside photo overlay should now work properly again (#9230)

## ~~Maxar Premium~~

One unfortunate news is that the _Maxar Premium_ imagery layer had to be removed from the background layers list for the time being, because the service was recently disabled by Maxar. As one can see by the long [discussion thread](https://community.openstreetmap.org/t/maxar-imagery-not-working-was-maxar-is-blurred-in-id-and-other-similar-topics/100524/30) on the OSM community forum, this is a loss of a very valuable source for mapping, especially in regions in developing countries where other imagery layers are not available, more outdated, or generally of lesser quality. That said, we are currently in talks with Maxar behind the scenes to see if and how the service could be restored for OSM mapping at some time in the (hopefully near) future.

<hr/>

The full changelog of this release is available at https://github.com/openstreetmap/iD/blob/release/CHANGELOG.md#2260.

<em>This discussion was created from the release <a href='https://github.com/openstreetmap/iD/releases/tag/v2.26.0'>v2.26.0</a>.</em>

**Comment by Zaczero** [comment]
<3 Omg! Omg!

My top new features:
* addr:place support
* bing streetside fixed

However, there are some critical issues that need fixing:
* https://github.com/openstreetmap/iD/issues/9762
* https://github.com/openstreetmap/iD/issues/9764

These sound like some minor bugs.

Anyway, thank you for such an amazing release!! :rocket:

**Reply by tyrasd** [reply]
[v2.26.2](https://github.com/openstreetmap/iD/blob/release/CHANGELOG.md#2262) has a fix for the raw tag editor bug
